scripts/add-new-version.js

```diff
@@ -1,51 +1,75 @@
 /**
- * This script adds a new version of the spec, by copying the latest one as baseline.
+ * This script adds a new version of the spec with examples, by copying the latest one as baseline.
  */
 const exec = require('child_process').exec;
 const fs = require('fs');
 const inputNewVersion = process.env.newVersion;
 //Regex taken from https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
-const versionRegex = /^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/g; //NOSONAR
+const versionRegex =
+	/^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/g; //NOSONAR
 
 /**
- * Promise based function to execute commands 
- * 
- * @param {string} command to execute 
- * @returns 
+ * Promise based function to execute commands
+ *
+ * @param {string} command to execute
+ * @returns
  */
 function execute(command) {
-  return new Promise((resolve, reject) => {
-    exec(command, function (error, stdout, stderr) { //NOSONAR
-      if (!error) resolve(stdout);
-      console.error(stderr);
-      reject(error);
-    });
-  });
+	return new Promise((resolve, reject) => {
```

Comment:
at the risk of starting a tabs-vs-spaces debate, I'm not sure it was ideal to reformat this file